flusher/flusher/handler.py

```diff
@@ -95,6 +101,15 @@ def handle_set_account(self, msg):
                 condition = (col == msg[col.name]) & condition
             self.conn.execute(accounts.update().where(condition).values(**msg))
 
+    def handle_set_interchain_account(self, msg):
```

Comment:
Do you expect to have set account later? if not should be new_interchain_account to create both row in accounts and interchain_account?

Reply:
emitter will write for both accounts and interchain_accounts

Reply:
already update this function to create both.